include/indy_mod.h

```diff
@@ -93,6 +93,9 @@ typedef enum
     // Attempt to send transaction without the necessary privileges
     LedgerSecurityError = 305,
 
+    // Attempt to create pool with name used for another exists pool
+    PoolLedgerAlreadyExistsError = 306,
```

Comment:
I am not sure that it is the correct terminology. Actually we are creating pool ledger config, but not just pool ledger. I suggest to change it in all places.